client/src/components/FourOhFour/index.js

```diff
@@ -47,7 +47,7 @@ class NotFoundPage extends Component {
             </blockquote>
           </div>
         ) : (
-          <Spinner color='#006400' name='ball-clip-rotate-multiple' />
+          <Spinner color='#6B6D6F' name='ball-clip-rotate-multiple' />
```

Comment:
This has been updated to a newer style in the Loaded component. You should make the loader component generic and use it where ever needed